src/notebook/Cell.jl

```diff
@@ -23,11 +23,20 @@ Base.@kwdef mutable struct Cell
     last_run_timestamp::Float64=0
     "Whether `this` inside `<script id=something>` should refer to the previously returned object in HTML output. This is used for fancy animations. true iff a cell runs as a reactive consequence."
     persist_js_state::Bool=false
+
+    # information to display cell dependencies
+    # Open: move to another place?
+    downstream_cells_map::Dict{Symbol,Vector{Cell}}=Dict{Symbol,Vector{Cell}}()
+    upstream_cells_map::Dict{Symbol,Vector{Cell}}=Dict{Symbol,Vector{Cell}}()
+    precedence_heuristic::Int=99
+
```

Comment:
@fonsp is it fine to keep this information in the Cell struct or should it be moved to somewhere else (e.g. to the new `ExprAnalysisCache`)?

Reply:
I put these information into the immutable struct `cell_dependencies`, resembling the structure in JS.